### PR TITLE
Graph total bootstrap time

### PR DESCRIPTION
**Pull request by tgnottingham**

### CODE REVIEW

**Comment by tgnottingham** [comment]
This adds the chart below the existing one (for crates with build times >= 30 seconds). I could integrate the two charts into one, using different scales/axes for the per-crate and total build times, if that seems better.

Also, for ease of implementation, this removes stripping out leading commits on the bootstrap charts which don't have relevant bootstrap data. Let me know if that's a problem. Didn't seem like a huge deal, especially since we're likely to have the bootstrap data for the commits we show when you land on the bootstrap page.

File: database/src/pool.rs [hunk 1]
```diff
@@ -67,8 +67,22 @@ pub trait Connection: Send + Sync {
         krate: &str,
         value: Duration,
     );
-
-    async fn get_bootstrap(
+    // Returns vector of bootstrap build times for the given artifacts. The kth
```
Comment (rylev) [inline database/src/pool.rs:70]: Nit: can we make these proper doc items instead of comments?

**Review by rylev — APPROVED** [review]
Looks good! I didn't have a chance to run this though. Can you make sure to post screenshots of any UI changes for review purposes? Thanks!!

**Comment by tgnottingham** [comment]
Thanks for the review!

Screenshot based on mock data:

bootstrap

Mock data, using two collections per artifact to test that it's correctly using the minimal time across all collections:

```
sqlite> select * from artifact;
id|name|date|type
1|test||release
2|sha01|1628366155|master
3|sha02|1628452555|master
4|sha03|1628538955|master

sqlite> select * from collection;
id|perf_commit
1|f9fa93720dbe5bd284204e7a0a467d257100f690
2|f9fa93720dbe5bd284204e7a0a467d257100f690
3|f9fa93720dbe5bd284204e7a0a467d257100f690
4|f9fa93720dbe5bd284204e7a0a467d257100f690
5|f9fa93720dbe5bd284204e7a0a467d257100f690
6|f9fa93720dbe5bd284204e7a0a467d257100f690
7|f9fa93720dbe5bd284204e7a0a467d257100f690

sqlite> select * from rustc_compilation;
aid|cid|crate|duration
2|2|rustc_middle|60000000000
2|2|rustc_mir|30000000000
2|2|rustc_index|5000000000
2|3|rustc_middle|65000000000
2|3|rustc_mir|35000000000
2|3|rustc_index|5000000000
3|4|rustc_middle|59000000000
3|4|rustc_mir|32000000000
3|4|rustc_index|5000000000
3|5|rustc_middle|65000000000
3|5|rustc_mir|35000000000
3|5|rustc_index|5000000000
4|6|rustc_middle|65000000000
4|6|rustc_mir|30000000000
4|6|rustc_index|5000000000
4|7|rustc_middle|60000000000
4|7|rustc_mir|38000000000
4|7|rustc_index|5000000000
```

**Review by rylev — APPROVED** [review]
Thanks! ♥

**Pull request merged**